lib/lightning_web/live/tokens_live/index.html.heex

```diff
@@ -0,0 +1,127 @@
+                  class="sticky right-1 w-12 h-12 rounded-full bg-blue-100 hover:bg-blue-100"
+                  id="copy"
+                  data-to="#new_token"
+                  phx-hook="Copy"
+                  phx-click="copy"
+                >
+                  <Heroicons.clipboard class="h-4 w-4 mx-auto" />
+                </button>
+              </div>
+            </div>
+          </.td>
+        </.tr>
+      <% end %>
+      <.tr>
+        <.th>Tokens</.th>
+        <.th class={
+          if Enum.empty?(@tokens) do
+            "text-right"
+          end
+        }>
+          <span class="px-4">Created at</span>
+        </.th>
+      </.tr>
+      <%= if Enum.empty?(@tokens) do %>
+        <.tr>
+          <.td>
+            <div class="flex content-center gap-2">
+              <div>
+                <Heroicons.finger_print outline class="h-12 w-12 text-black" />
+              </div>
+              <div>
+                <h3 class="mt-1 text-sm font-semibold text-gray-900">
+                  No Personal Access Tokens
+                </h3>
+                <p class="mt-1 text-sm text-gray-500">
+                  Get started by creating a new access token.
+                </p>
+              </div>
+            </div>
+          </.td>
+          <.td>
+            <div class="my-auto p-4 text-right">
+              <Common.button
+                phx-click="generate_new_token"
+                id="generate_new_token"
+              >
+                <div class="h-full">
+                  <Icon.plus class="h-4 w-4 inline-block" />
+                  <span class="inline-block align-middle">
+                    Generate New Token
+                  </span>
+                </div>
+              </Common.button>
+            </div>
+          </.td>
+        </.tr>
+      <% end %>
+      <%= for token <- @tokens do %>
+        <.tr id={"token-#{token["id"]}"}>
+          <.td>
+            <div class="flex rounded overflow-auto p-2 bg-blue-100 w-auto justify-between">
+              <code class="my-auto whitespace-nowrap py-2 px-4 align-baseline text-base font-medium leading-none">
+                <span>
+                  <%= token["token"] %>
+                </span>
+              </code>
+            </div>
+          </.td>
+          <.td>
+            <div class="flex content-center justify-between">
+              <span class="my-auto whitespace-nowrap py-2 px-4 text-center align-baseline font-medium leading-none">
+                <%= token["inserted_at"] %>
+              </span>
+              <%= link(
+                  to: "#",
+                  phx_click: "delete_token",
+                  phx_value_id: token["id"],
+                  data: [
+                    confirm:
+                      "Any applications or scripts using this token will no longer be able to access the API. You cannot undo this action. Are you sure you want to delete this token?"
+                  ]
+                ) do %>
```

Comment:
We should be using the modal component here for this.

I'll leave this here for now, I'll refer to @amberrignell to make a call.

Reply:
Added a `token_deletion_modal.ex` for token deletion